guides/contributor/donations.md

```diff
@@ -0,0 +1,60 @@
+# Donations
+
+Donations of preexisting code fall into two broad categories:
+
+* **Small donations:** Some donations only amount to a single PR and should
```

Comment:
nit: I like anchoring to one PR, but reality of instrumentation PRs is that they are often multiple and over time. Not sure the better word, but isolated feature comes to mind. Like, someone shouldn't need to think they have to go through a donation process for net new code that has no existing codebase or license, or is independent from one similar. Think about httpclient v5 being different than v4 maybe needing multiple PRs, but in act is quite a routine thing and not really like a donation sort of affair.

My 2p

Reply:
going to merge this now, raise a follow up issue for further discussion